File: 2.server/src/main/java/ru/vitaliy/petrov/server/forms/requests/UserProfileUpdateRequest.java
package ru.vitaliy.petrov.server.forms.requests;

import lombok.Data;

@Data
public class UserProfileUpdateRequest {
    private final String updatedFullName;
    private final String updatedDateOfBirth;
    private final String updatedResidentialAddress;
    private final String updatedPlaceOfWork;
    private final String updatedPositionAtWork;
    private final String updatedWorkPhoneNumber;
    private final String updatedDriverLicenseNumber;
}

File: 3.client/app/src/main/java/com/petrov/vitaliy/caraccidentapp/domain/models/requests/user/UserProfileUpdateRequest.kt
package com.petrov.vitaliy.caraccidentapp.domain.models.requests.user

import java.sql.Date

data class UserProfileUpdateRequest(
    val updatedFullName: String,
    val updatedDateOfBirth: Date,
    val updatedResidentialAddress: String,
    val updatedPlaceOfWork: String,
    val updatedPositionAtWork: String,
    val updatedWorkPhoneNumber: String,
    val updatedDriverLicenseNumber: String
)
